google-cloud-speech/acceptance/speech/async_test.rb

```diff
@@ -166,4 +166,21 @@
     results.first.confidence.must_be_close_to 0.9, 0.1
     results.first.alternatives.must_be :empty?
   end
+
+  it "recognizes audio with words" do
+    op = speech.process filepath, encoding: :linear16, language: "en-US", sample_rate: 16000, words: true
```

Comment:
@quartzmo I think Phil's point here is, pretty soon, developers will have to write `enable_word_time_offsets` soon, so they should write it now to ease migration.